cmd/containerimage/one.go

```diff
@@ -54,6 +60,13 @@ func replaceOne(cmd *cobra.Command, args []string) error {
 
 	img := r.Context().String()
 
+	if len(platform) != 0 {
+		if !strings.Contains(platform, "/") {
+			return fmt.Errorf("platform must be in the format os/arch")
+		}
+		os, arch := strings.Split(platform, "/")[0], strings.Split(platform, "/")[1]
+		containers.SetPlatform(&gocrv1.Platform{OS: os, Architecture: arch})
+	}
 	sum, err := containers.GetDigest(ctx, ref)
```

Comment:
I'd rather pass the platform as a parameter to GetDigest (with a default Platform struct keeping the current behavior) than having `SetPlatform` set a global parameter in the `containers` module.